verification.md

```diff
@@ -0,0 +1,58 @@
+# ✅ Verification
+
+In this section, you'll play the role of a <span class="label-role verifier">verifier</span>: 
+* You'll take an `AttestedClaim` object;
+* You'll verify that its data is correct;
+* You'll verify that it exists, aka that it's on-chain.
+
+## Get an `AttestedClaim` object
+ 
+You can either:
+* take the `AttestedClaim` object you've generated in the previous step as an <span class="label-role attester">attester</span>;
+* or select one from this [shared doc](https://hackmd.io/c6OBNgWWR8yWJhMj7WICUA?edit);
+* or if you're in a workshop, ask another participant to send you their `AttestedClaim` object via email/another messaging system.  
```

Comment:
I think to remove the end of this sentence "via messaging and email".

Reply:
Sure, done - is that because you find this bit too long or confusing (or both)?